spec/integration/job_spec.rb

```diff
@@ -169,7 +169,7 @@ class NoPerformJob; end
       queue.put('Foo', {}, jid: 'c', depends: ['a'])
       expect(client.jobs['c'].dependencies).to eq(['a'])
       client.jobs['c'].depend('b')
-      expect(client.jobs['c'].dependencies).to eq(%w{a b})
+      expect(client.jobs['c'].dependencies).to match_array(%w{a b})
```

Comment:
Was failing before my changes. Looks like the order of dependacis my not be guaranteed.